redis-persistence/src/main/java/com/netflix/conductor/dyno/DynomiteConfiguration.java

```diff
@@ -0,0 +1,65 @@
+package com.netflix.conductor.dyno;
+
+import com.netflix.conductor.core.config.Configuration;
+
+public interface DynomiteConfiguration extends Configuration {
+    // FIXME Are cluster and cluster name really different things?
+    String CLUSTER_PROPERTY_NAME = "workflow.dynomite.cluster";
+    String CLUSTER_DEFAULT_VALUE = null;
+
+    String CLUSTER_NAME_PROPERTY_NAME = "workflow.dynomite.cluster.name";
+    String HOSTS_PROPERTY_NAME = "workflow.dynomite.cluster.hosts";
+
+    String MAX_CONNECTIONS_PER_HOST_PROPERTY_NAME = "workflow.dynomite.connection.maxConnsPerHost";
+    int MAX_CONNECTIONS_PER_HOST_DEFAULT_VALUE = 10;
+
+    String ROOT_NAMESPACE_PROPERTY_NAME = "workflow.namespace.queue.prefix";
+    String ROOT_NAMESPACE_DEFAULT_VALUE = null;
+
+    String DOMAIN_PROPERTY_NAME = "workflow.dyno.keyspace.domain";
+    String DOMAIN_DEFAULT_VALUE = null;
+
+    String NON_QUORUM_PORT_PROPERTY_NAME = "queues.dynomite.nonQuorum.port";
+    int NON_QUORUM_PORT_DEFAULT_VALUE = 22122;
+
+    default String getCluster() {
```

Comment:
**Question**: Is the use of package scope here intended? The absence of `public`, `protected` and `private` leaves these only accessible within the package.

Reply:
You might think that, but `interface`s are special, the default is `public` which is indeed what we want.